src/qt/clientmodel.cpp

```diff
@@ -288,7 +289,7 @@ static void BlockTipChanged(ClientModel* clientmodel, SynchronizationState sync_
     const bool throttle = (sync_state != SynchronizationState::POST_INIT && !fHeader) || sync_state == SynchronizationState::INIT_REINDEX;
     const int64_t now = throttle ? GetTimeMillis() : 0;
```

Comment:
Could we do the following change?
```suggestion
    const auto now = throttle ? GetTime<std::chrono::milliseconds>() : 0ms;
```

Reply:
We can't do this because we want sys time not mockable time

Reply:
We don't want to do this because we want sys time, not mockable time.

Reply:
> we want sys time, not mockable time

That makes sense. Thanks for the clarification.